src/test/java/systemtests/HelpCommandSystemTest.java

```diff
@@ -59,6 +59,10 @@ public void openHelpWindow() {
         executeCommand(HelpCommand.COMMAND_WORD);
         getMainWindowHandle().focus();
 
+        // open help window and give it focus using help alias
+        executeCommand(HelpCommand.COMMAND_ALIAS);
```

Comment:
This test does nothing. This is not actually testing whether the command alias works. Please try to understand what the tests are doing instead of blindly copy and pasting :(